recipes/python-pathlib2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: python-pathlib2
+  version: {{ version }}
+
+source:
+  fn: pathlib2-{{ version }}.tar.gz
+  url: https://github.com/mcmtroffaes/pathlib2/archive/{{ version }}.tar.gz
+  md5: 5ef144f5050f17962e9cef35c0e93ed7
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - pathlib2
+
+about:
+  home: https://github.com/mcmtroffaes/pathlib2
+  license: MIT
+  summary: "Fork of pathlib aiming to support the full stdlib Python API."
+
+extra:
+  recipe-maintainers:
+    - pelson
+    - takluyver
```

Comment:
Meant to ping you on this @takluyver - I'm trying to increase the bus factor on the conda recipe so added you to the maintainers. Please do let me know if that isn't OK.